clickhouse/datadog_checks/clickhouse/clickhouse.py

```diff
@@ -0,0 +1,104 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import clickhouse_driver
+from six import raise_from
+
+from datadog_checks.base import AgentCheck, is_affirmative
+from datadog_checks.base.utils.db import QueryManager
+
+from . import queries
+from .utils import ErrorSanitizer
+
+
+class ClickhouseCheck(AgentCheck):
+    __NAMESPACE__ = 'clickhouse'
+    SERVICE_CHECK_CONNECT = 'can_connect'
+
+    def __init__(self, name, init_config, instances):
+        super(ClickhouseCheck, self).__init__(name, init_config, instances)
+
+        self._server = self.instance.get('server', '')
+        self._port = self.instance.get('port')
+        self._db = self.instance.get('db', 'default')
+        self._user = self.instance.get('user', 'default')
+        self._password = self.instance.get('password', '')
+        self._connect_timeout = float(self.instance.get('connect_timeout', 10))
+        self._read_timeout = float(self.instance.get('read_timeout', 10))
+        self._compression = self.instance.get('compression', False)
```

Comment:
Missing `is_affirmative` ?

Reply:
This actually can be a string https://clickhouse-driver.readthedocs.io/en/latest/api.html#connection